dashboard/pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -113,6 +151,11 @@ export default Vue.extend<any, any, any, any>({
         this.sectionContent[2].isLoaded = true;
         this.sectionContent[2].isEnable = true;
       }
+    },
+    openMetricsDetails() {
+      const clusterMetricsLink = `${ window.location.origin }/c/${ this.$store.getters['clusterId'] }/explorer#cluster-events`;
+
+      window.open(clusterMetricsLink, '_blank');
```

Comment:
this should use $router. also as we're in the dashboard context i think it's find to open in the same tab

Reply:
this one is still remaining to do